reference/configuration/tidb-server/configuration-file.md

```diff
@@ -254,6 +254,12 @@ Configuration items related to performance.
 - Default value: `5000`
 - If a transaction does not roll back or commit after the number of statements exceeds `stmt-count-limit`, TiDB returns the `statement count 5001 exceeds the transaction limitation, autocommit = false` error. This configuration takes effect **only** in the retriable optimistic transaction. If you use the pessimistic transaction or have disabled the transaction retry, the number of statements in a transaction is not limited by this configuration.
 
+### `txn-total-size-limit`
+
+- The size limit of a transaction.
+- Default value: `104857600`
```

Comment:
What about adding a time unit?

Reply:
In the description, it says "size in bytes"  @TomShawn